client/src/pages/product.jsx

```diff
@@ -0,0 +1,100 @@
+import Header from '../components/Header';
+import NavBar from '../components/NavBar';
+import styled from 'styled-components';
+import { useParams } from 'react-router-dom';
+
+function Product({ tastes }) {
+  const { id } = useParams();
+  const taste = tastes.find(taste => taste.id === id);
+
```

Comment:
What happens if the `id` is not found? I highly recommend adding a check that returns a visible info for the user, that the `id` was not found.

Reply:
Hey Martin! Thank you again for your valuable feedback! The error is now managed, please confirm so I can merge. Thanks!